verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -107,104 +190,107 @@ class NewCaseForm extends React.Component<Props, NewCaseFormState> {
         const ageRange = values.age
             ? { start: values.age, end: values.age }
             : { start: values.minAge, end: values.maxAge };
-        try {
-            await axios.post('/api/cases', {
-                demographics: {
-                    sex: values.sex,
-                    ageRange: ageRange,
-                    ethnicity: values.ethnicity,
-                    nationalities: values.nationalities,
-                    profession: values.profession,
+        const newCase = {
+            demographics: {
+                sex: values.sex,
+                ageRange: ageRange,
+                ethnicity: values.ethnicity || '',
```

Comment:
remove `|| ''` here and for professions too?

Reply:
yup, done.